examples/oneapi/dpc/source/kmeans_init/kmeans_init_default_dense.cpp

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <CL/sycl.hpp>
+#include <iomanip>
+#include <iostream>
+
+#define ONEAPI_DAL_DATA_PARALLEL
+#include "oneapi/dal/algo/kmeans_init.hpp"
+#include "oneapi/dal/data/accessor.hpp"
+
+#include "example_util/utils.hpp"
+
+using namespace oneapi;
+
+void run(sycl::queue &queue) {
+  std::cout << "Running on "
```

Comment:
Align example with others examples for dpcpp

Reply:
Why resolved? See example for other algorithms, for example: https://github.com/oneapi-src/oneDAL/blob/master/examples/oneapi/dpc/source/knn/knn_cls_brute_force_dense_batch.cpp#L87

Reply:
fixed